themes/digital.gov/src/scss/new/_card-event.scss

```diff
@@ -86,20 +181,92 @@
 // Upcoming events
 // ==========================================================================
 .card-event-promo {
-  grid-template-areas:
-  "c-date c-content"
-  "c-meta c-meta" !important;
-  grid-template-columns: 1fr 5fr !important;
-
-  .card__media {
-    @include u-border-right("1px", "indigo-warm-80", "solid");
-    grid-area: c-date !important;
-  }
 }
 
-//
 // Past events
 // ==========================================================================
 .card-event-past {
-  border-top: none;
+  @include u-border-top(0);
```

Comment:
Consider keepting the border, but making the color transparent to avoid a shift in layout.

Reply:
I've removed border in [f4ece54](https://github.com/GSA/digitalgov.gov/pull/6021/commits/f4ece543373aec364248d25f530ae7aab85ff118).